### DISCUSSION TITLE
TFLite Conversion of db_resnet50

### DISCUSSION THREAD

**Discussion by SavvaVyatkin** [mindee/doctr#1255] [Q&A]
How do I convert the db_resnet50 to tflite? I have only been able to find instructions for how to convert a single layer to tflite, which is not very useful.

Furthermore, when I try to convert the db_resnet50 in this fashion

`import tensorflow as tf
from doctr.models import db_resnet50

model = db_resnet50(pretrained=True)

tf.saved_model.save(model, 'db_resnet50')
imported = tf.saved_model.load('db_resnet50')
print(imported.signatures)

converter = tf.lite.TFLiteConverter.from_saved_model('db_resnet50')
converter.optimizations = [tf.lite.Optimize.DEFAULT]
converter.target_spec.supported_types = [tf.float16]
tflite_quant_model = converter.convert()
tflite_quant_model.save('db_resnet50.tflite')
`

I get

`ValueError: Only support at least one signature key.`

And printing out the signatures gets

`_SignatureMap({})`

**Comment by felixT2K** [comment]
Hi @SavvaVyatkin 👋🏼 ,

seems you was able to solve it on your own ? :)

Sorry for the confusion, I know there is another page in the documentation about the TFLite export. 
However, this is outdated and will be completely removed / replaced in the next release. 
We have decided to support ONNX as an export format because it is much more dynamic and works for TF and PT, which is why TFLite and SavedModel are no longer officially supported by us.
